### ISSUE TITLE
What is the use of sysctl.d/99-powersave.conf for an SSD drive?

### ISSUE THREAD

**Issue by adrfantini** [Hjdskes/powersave#4]
Most of the content of `sysctl.d/99-powersave.conf` refer to delaying disk writes, which should be, if I'm not mistaken, not necessary on an SSD drive. Is this wrong? Did you do any testing?

Thanks

**Comment by Hjdskes** [comment]
That depends on what you mean with "not necessary", I suppose. If your goal is power saving, then you are probably correct in that it does not save as much power on an SSD as it does on an HDD (haven't tested how much of a difference, if any, it makes. I assume it would be negligible). However, less writing improves the lifespan of your SSD so in that sense you could deem it necessary (although, with modern SSDs, this is also not much of an issue anymore).

In any case, it won't hurt to have them listed. Most of the entries in that file are currently commented, too.

**Comment by adrfantini** [comment]
Still it wouldn't be bad to specify that the powersaving impact of those
changes are probably minimal for SSDs. I would think that with a modern SSD
those settings are probably best left at the default values as the benefits
are very limited, and you pay a small price in responsiveness and amount of
data lost on a crash.

Thanks for the good job, by the way. Your tweaks are very comprehensive and
useful.

2017-01-27 11:50 GMT+01:00 Jente Hidskes <notifications@github.com>:

**Comment by Hjdskes** [comment]
> Still it wouldn't be bad to specify that the powersaving impact of those
changes are probably minimal for SSDs.

I'm planning on writing an in-depth blog post about these settings, in which I will definitely mention this. I'll add it to the README too.

> I would think that with a modern SSD those settings are probably best left at the default values as the benefits are very limited, and you pay a small price in responsiveness and amount of data lost on a crash.

They *are* left at the defaults, except for `vm.dirty_writeback_centisecs` (which, I admit, is a leftover from the time where I did not have an SSD - I'll probably change it when I write that blog post). In this case I'd still like to keep them in, if only to notify others of their existence. I have not noticed a (however small) decrease in responsiveness nor (increased) data loss on a system crash. In any case, feel free to change the settings to whatever you prefer!

**Comment by adrfantini** [comment]
Yeah, of course. Just wanted to make sure that it was clear that on SSDs
the settings do not change much.

All the best, thanks!

2017-01-28 15:44 GMT+01:00 Jente Hidskes <notifications@github.com>:

**Comment by Hjdskes** [comment]
No problem, I appreciate the message!
